spec/models/spree/admin/tabs/root_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+module Spree
+  module Admin
+    describe Tabs::Root, type: :model do
+      let(:root) { described_class.new }
+
+      describe '#add(item)' do
```

Comment:
Tiny detail, but do we have any other specs that use this convention (`#method_name(params)`)? I think it's usually just the method name